modules/apps/kate/default.nix

```diff
@@ -196,25 +196,65 @@ in
     '';
   };
 
+  # ==================================
+  #     BRACKETS
+  options.programs.kate.editor.brackets = {
+    characters = lib.mkOption {
+      type = lib.types.str;
+      default = "<>(){}[]'\"\`";
+      example = "<>(){}[]'\"\`*_~";
+      description = "Which characters kate will treat as brackets.";
+    };
+    automaticallyAddClosing = lib.mkEnableOption {
+      description = "that a closing bracket is automatically inserted upon typing the opening";
```

Comment:
Capitalization and punctuation

Reply:
What do you mean?
`programs.kate.editor.brackets.characters` is defined via `mkOption`, which means, I have to start (by capitalizing) and end (by punctuation) the sentence myself. Which isn't the case for all the other options, that are defined with `mkEnableOption`.